_layouts/member.html

```diff
@@ -47,6 +47,17 @@
 		</div>
 		<div class="bigspacer"></div>
 		{% endif %}
+		{% if member.fediverse %}
+		<div class="smallhead">
+			Fediverse
+		</div>
+		<div class="pad-left note">
+			<div class="smallspacer"></div>
+			<i class="fa fa-handshake-o"></i>
+			<a class="off" href="https://{{ member.fediverse.server }}/{{ member.fediverse.handle }}">{{ member.fediverse.handle }}</a>
+		</div>
+		<div class="bigspacer"></div>
+		{% endif %}
```

Comment:
> Also completely open to a decision that this is not something y'all want to support, in which case I'll send in a new PR sans fediverse stuff.

I don't have a problem with this - it doesn't affect any other member pages and looks fine on your page.

> I'm not sure if there's some documentation of supported entries in the team section front matter that I should also update; please point me at it if there is.

Not that I know of. I'm assuming most people have just looked at existing pages for inspiration.